docs/index.html

```diff
@@ -195,7 +195,7 @@ <h4 class='speakers__title'>Speakers:</h4>
 </li>
 <li class='speakers__item'>
 <a class='speakers__link' href='https://twitter.com/jmcharnes' target='_blank'>
-<img alt='Jason Charnes' src='./assets/images/speakers/Jason-Charnes.jpg'>
+<img alt='Jason Charnes' src='./assets/images/speakers/Jason-charnes.jpg'>
```

Comment:
I think the change to image files shouln't be here, seems like the filenames are currently correct: https://github.com/RubyConfCo/rubyconfco-website-2017/tree/master/docs/assets/images/speakers